tests/layer_tests/pytorch_tests/test_set_item.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class TestSilu(PytorchLayerTest):
+    def _prepare_input(self):
+        import numpy as np
+        return [np.random.randn(10).astype(np.int32).tolist()]
+
+    def create_model(self, idx):
+        import torch
+        from typing import List
+
+        class aten_set_item(torch.nn.Module):
+            def __init__(self, idx):
+                super(aten_set_item, self).__init__()
+                self.idx = idx
+
+            def forward(self, x: List[int]):
+                x[self.idx] = 0
+                return x
```

Comment:
otherwise number of outputs is different due to list flattening
```suggestion
                return torch.tensor(x)
```